src/deprecations.jl

```diff
@@ -26,6 +26,7 @@ ones(T::Type, dims...) = Base.ones(T, dims...) # no need for a message
 
 function zeros(dims...)
   Base.depwarn("Flux.zeros(size...) is deprecated, please use Flux.zeros32(size...) or Base.zeros(Float32, size...)", :ones)
```

Comment:
```suggestion
  Base.depwarn("Flux.zeros(size...) is deprecated, please use Flux.zeros32(size...) or Base.zeros(Float32, size...)", :zeros)
```
Not sure what that is